sfdc/Module.md

```diff
@@ -368,178 +241,37 @@ their organization. Following are the remote functions available for retrieving
-
-## [Samples for Salesforce REST API use cases](samples/rest_api_usecases)  
-These samples demonstrate the employment of Ballerina Salesforce Connector in Salesforce REST API related operations. 
-The samples can be further divided as following
-* Samples that can be used with any SObject's CRUD operations
-* Samples for convenient access of Account, Contact, Product, Opportunity and Target SObjects's CRUD operations
-* Samples for SOSL and SOQL related operations
-* Samples for retrieving Organization and SObject metadata
-
-## [Samples for Salesforce Bulk API use cases](samples/bulk_api_usecases)
-These samples demonstrate the employment of Ballerina Salesforce Connector in Salesforce BULK API related operations. 
-Examples for bulk insert, bulk insert through files, bulk update, bulk upsert and bulk delete using json, csv or xml 
-data sets are given here.
-
-## [Samples for Event Listener](samples/event_listener_usecases)
-This sample demonstrates on capturing events using the Event Listener of Ballerina Salesforce Connector. As mentioned 
-above to listen to a certin event users need to select Objects for Change Notifications in the user interface in his/her Salesforce instance. 
-- [Select SObjects](https://developer.salesforce.com/docs/atlas.en-us.change_data_capture.meta/change_data_capture/cdc_select_objects.htm)
-# Building from the Source
-## Setting up the prerequisites
-*   Download and install Java SE Development Kit (JDK) version 11 (from one of the following locations).
-	[Oracle](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html),
-	[OpenJDK](https://adoptopenjdk.net/)  
-
-    **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
-
-*   Download and install[ Ballerina SL Alpha5](https://ballerina.io/).
-*   Install Gradle  
-
-## Building the Source
-Execute the commands below to build from the source after installing the Ballerina SL Alpha5 version.
-
-### Build the project:
-```ballerina
-   gradle build
-```
-
-### To build the library:
-```ballerina
-   bal build ./sfdc
-```
-
-### To build the module without the tests:
-```
-   bal build --skip-tests ./sfdc
-```
-
-## Contribution to Ballerina
-As an open source project, Ballerina welcomes contributions from the community.  
-For more information, go to the [contribution guidelines](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md).  
-
-## Code of Conduct
-All the contributors are encouraged to read the Ballerina [Code of Conduct](https://ballerina.io/code-of-conduct).
-
-## Useful Links
-* Discuss the code changes of the Ballerina project in [ballerina-dev@googlegroups.com](mailto:ballerina-dev@googlegroups.com).
-* Chat live with us via our [Slack channel](https://ballerina.io/community/slack/).
-* Post all technical questions on Stack Overflow with the [#ballerina tag](https://stackoverflow.com/questions/tagged/ballerina).
+  ```ballerina
```

Comment:
Shouldn't this be a full example?

Reply:
Added in https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/156/commits/e4b1d0a837a4ba241b6f7ece1a4b730aa64222e5